src/data/mod.rs

```diff
@@ -2,6 +2,7 @@
 
 pub use self::arena::{Arena, Index};
 pub use self::coarena::Coarena;
+pub use graph::{Externals, Neighbors, NodeReferences};
```

Comment:
Not entirely sure about this. it seems practical.

Another way to expose it would be through the graph module, but the `pub(crate) mod graph;` means that we'd have to change all public fields within `graph` to `pub(crate)`

Reply:
Let’s remove the dead code. We can revive it from the git history if we really need it in the future.